synapse/lib/agenda.py

```diff
@@ -270,7 +271,7 @@ class _Appt:
         'lastfinishtime',
     }
 
-    def __init__(self, stor, iden, recur, indx, query, creator, recs, nexttime=None, view=None):
+    def __init__(self, stor, iden, recur, indx, query, creator, recs, nexttime=None, view=None, created=None):
```

Comment:
Is it time to just pass in the cdef dict rather than having a zillion args?

Reply:
In 3.0 yea. I was shooting for this to land in 2.0, so I didn't want to make any sweeping API changes. But it'd be nice for 3.0 to make this a documented cdef dict instead of this.